src/_utils/TradeUtils.js

```diff
@@ -20,3 +20,50 @@ export const digitOptions = (from, to) => {
     const arr = Array.from(tArr);
     return arr.map((i, idx) => idx + from).map(x => ({ value: x, text: x.toString() }));
 };
+
+export const durationText = unit => {
+    switch (unit) {
+        case 't': return 'Ticks';
+        case 's': return 'Seconds';
+        case 'm': return 'Minutes';
+        case 'h': return 'Hours';
+        case 'd': return 'Days';
+        default: return null;
```

Comment:
ok ~ should we avoid null at all? I dont really understand the intention of having both